ceremonies/rpc/src/lib.rs

```diff
@@ -66,13 +137,27 @@ where
 	) -> Result<Vec<(CeremonyIndexType, CommunityReputation)>> {
 		self.deny_unsafe.check_if_safe()?;
 
-		let api = self.client.runtime_api();
-		let at = BlockId::hash(at.unwrap_or_else(|| self.client.info().best_hash));
-		return Ok(api.get_reputations(&at, &account).map_err(runtime_error_into_rpc_err)?)
+		if !self.offchain_indexing {
+			return Err(offchain_indexing_disabled_error("ceremonies_getReputations"))
+		}
+
+		if self.cache_dirty(
+			[REPUTATION_CACHE_DIRTY_KEY, account.encode().as_slice()].concat().as_slice(),
```

Comment:
Why do you derive the reputation cache key, and the dirty bit cache key in two different ways?

Anyhow, please use a function as I said above.